android-libs/src/androidTest/java/io/ooni/mk/androidTest/MKWebConnectivityTest.java

```diff
@@ -3,26 +3,33 @@
 import org.junit.Test;
 
 import androidx.test.filters.SmallTest;
+import androidx.test.core.app.ApplicationProvider;
 
 import com.google.gson.Gson;
 
 import java.util.ArrayList;
 
+import android.content.Context;
+import com.google.common.truth.Truth;
 import io.ooni.mk.MKAsyncTask;
+import io.ooni.mk.MKResourcesManager;
 
 @SmallTest public class MKWebConnectivityTest {
     static {
         System.loadLibrary("measurement_kit");
     }
 
     @Test public void perform() {
+        Context context = ApplicationProvider.getApplicationContext();
+        boolean okay = MKResourcesManager.maybeUpdateResources(context);
+        Truth.assertThat(okay).isTrue();
         MKSettings settings = new MKSettings();
         Gson gson = new Gson();
         settings.name = "WebConnectivity";
         settings.log_level = "INFO";
-        settings.options.ca_bundle_path = "cacert.pem";
-        settings.options.geoip_asn_path = "asn.mmdb";
-        settings.options.geoip_country_path = "country.mmdb";
+        settings.options.ca_bundle_path = MKResourcesManager.getCABundlePath(context);
+        settings.options.geoip_asn_path = MKResourcesManager.getASNDBPath(context);
+        settings.options.geoip_country_path = MKResourcesManager.getCountryDBPath(context);
         settings.inputs = new ArrayList<String>();
         settings.inputs.add("https://www.torproject.org/");
         settings.inputs.add("https://x.org/");
```

Comment:
We probably want to test the URL that was causing https://github.com/measurement-kit/measurement-kit/issues/1877